front/app/components/FormSync/ExcelDownloadButton.tsx

```diff
@@ -40,7 +41,12 @@ const ExcelDownloadButton = ({ phaseId }: Props) => {
         onClick={downloadExampleXlsxFile}
         disabled={!inputImporterAllowed}
       >
-        <FormattedMessage {...messages.downloadExcelTemplate} />
+        <Box display="flex" alignItems="center" gap="4px">
+          <FormattedMessage {...messages.downloadExcelTemplate} />
+          <IconTooltip
```

Comment:
Make sure the tooltip in the dropdown list item is not lost (and it now shows for "download Excel template" buttons everywhere).